templates/web-application/webpack.dev.js

```diff
@@ -0,0 +1,89 @@
+// @ts-check
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import ReactRefreshWebpackPlugin from "@pmmmwh/react-refresh-webpack-plugin";
+import open from "open";
+
+import { loadSwcConfig } from "./loadSwcConfig.js";
+
+/** @type {import("webpack").Configuration} */
+
+class OpenBrowser {
```

Comment:
This is new, it make the browser open on the right page once the webpack build is done.

Reply:
Is there a specific reason to use this instead of `devServer: { open: true }` ?